app/assets/stylesheets/application/donations/donations.scss

```diff
@@ -40,9 +40,34 @@
   width: 32%;
 }
 
+label[for="donation_amount_other"] {
+  margin-top: 1px;
+  vertical-align: middle;
+  width: 32%;
+  #donation_amount_other {
+    display: none;
+  }
+  input#other_text {
+    width: 100%;
+  }
+}
+
 .form-group.donation-amount .form-control {
   display: inline-block;
+  font-size: 24px;
+  padding: 25px 15px;
   width: 32%;
+  vertical-align: top;
+}
+
+.radio-label {
+  margin-bottom: 0;
+}
+
+#tip_other_text {
```

Comment:
Avoid using id selectors<br>Selector `tip_other_text` should be written in lowercase with hyphens